e2e/connect.test.tsx

```diff
@@ -45,6 +45,16 @@ describe('connect', () => {
       expect(toolbar.textContent).toBe('Connect wallet to swap')
     })
 
+    it('expects widget not to be disabled', async () => {
```

Comment:
should this be separated into separate tests? or it needs to happen in sync

Reply:
I'll put the singleton test in its own describe block